### ISSUE TITLE
Cannot create NTRU addresses in I2P-Bote 0.4.4 (Trac #1889)

### ISSUE THREAD

**Issue by str4d** [i2p/i2p.i2p-bote#85]
Reported via I2P-Bote :)

> When I'm trying to create an identity, there are 4 available encryption algorithms, ElGamal, P-256, P-521 and NTRU-1087. I have successfully created all algorithms except NTRU-1087. After I entered a name and picked NTRU-1087, clicked the "Create" button, an error message appeared immediately, said, "Form must be submitted using POST".

This was caused by the CSRF changes I implemented in 0.4.4 - it looks like I missed a form submission somewhere that still uses GET.

Migrated from https://trac.i2p2.de/ticket/1889
```json
{
    "status": "closed", 
    "changetime": "2017-01-08T14:43:53", 
    "description": "Reported via I2P-Bote :)\n\n> When I'm trying to create an identity, there are 4 available encryption algorithms, ElGamal, P-256, P-521 and NTRU-1087. I have successfully created all algorithms except NTRU-1087. After I entered a name and picked NTRU-1087, clicked the \"Create\" button, an error message appeared immediately, said, \"Form must be submitted using POST\".\n\nThis was caused by the CSRF changes I implemented in 0.4.4 - it looks like I missed a form submission somewhere that still uses GET.", 
    "reporter": "str4d", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1483886633490377", 
    "component": "apps/plugins", 
    "summary": "Cannot create NTRU addresses in I2P-Bote 0.4.4", 
    "priority": "minor", 
    "keywords": "I2P-Bote", 
    "version": "0.9.27", 
    "parents": "", 
    "time": "2016-12-11T06:37:31", 
    "milestone": "0.9.29", 
    "owner": "str4d", 
    "type": "defect"
}
```

**Comment by str4d** [comment]
Trac update at 20161211T06:37:47: 
* str4d changed owner from "" to "str4d"
* str4d changed status from "new" to "accepted"

**Comment by str4d** [comment]
Trac update at 20170108T14:43:53: 
* str4d commented:

> Fixed in `402f3bd645d81b9fc62ff5e22a45ca16f2801dbc`.


* str4d changed milestone from "soon" to "0.9.29"
* str4d changed resolution from "" to "fixed"
* str4d changed status from "accepted" to "closed"
